pulp_file/tests/functional/api/test_crud_remotes.py

```diff
@@ -128,6 +127,52 @@ def test_all(self):
             api.Client(config.get_config()).post(FILE_REMOTE_PATH, body)
 
 
+class RemoteDownloadPolicyTestCase(unittest.TestCase):
+    """Verify download policy behavior for valid and invalid values.
+
+    This test targets the following issues:
+
+    * `Pulp #4420 <https://pulp.plan.io/issues/4420>`_
```

Comment:
```suggestion
This test does the following:

1. Describe action...
2. Describe assertion...

    * `Pulp #4420 <https://pulp.plan.io/issues/4420>`_
```

Reply:
Agree. Added more comments to what is being tested and where more information can be found.